src/scripts/main.js

```diff
@@ -1,3 +1,320 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const tableRows = tableBody.rows;
+let toggle = true;
+let valueForSort = 0;
+let special;
+
+function setSalaryValue(num) {
+  return `$${num.toLocaleString('en-US')}`;
+};
+
+function getSalaryValue(str) {
```

Comment:
The function 'getSalaryValue' can be simplified using an arrow function: `const getSalaryValue = str => +str.replace(/[,$]/g, '');`